CSharp/DevVmPowershell/Helpers/Constants.cs

```diff
@@ -80,6 +80,8 @@ public class Waiting
 		{
 			public const int MAX_WAIT_TIME_IN_MINUTES = 10;
 			public const int SLEEP_TIME_IN_SECONDS = 15;
+			public const int RETRY_COUNT = 60;
```

Comment:
- [ ] If these values are specific to the imagingHelper waiting, I would put that in the variable name to be specific